conda/utils.py

```diff
@@ -7,6 +7,36 @@
 from os.path import abspath, isdir, join
 import os
 
+
+def can_open(file):
+    """
+    Return True if the given ``file`` can be opened for writing
+    """
+    try:
+        fp = open(file, "ab")
+        fp.close()
+        return True
+    except IOError:
```

Comment:
I would prefer to be more explicit here.  Catching the broader class of errors might cover up other problems that we're not anticipating.  For example, the try/except in `conda.install` covers up the permission error when all its there for is file not found exceptions.

Related -- I've seen issues in wakari where `IOError` was not caught by `OSError` when dealing with files that were not found or writable one.  Thought it was workbench, but doesn't appear to be the case.  Actually, it looks like this has changed between Python 2.7 and 3.3.  According to the [docs][1]:

> Changed in version 3.3: EnvironmentError, IOError, WindowsError, VMSError, socket.error, select.error and mmap.error have been merged into OSError.

But if you go back to the [hierarchy tree][2] on Python 2, it clearly shows `IOError` and `OSError` as siblings.  I have verified that `IOError` still works inside Python 3.4.3, though, so I think we're safe there.

[1]: https://docs.python.org/3/library/exceptions.html#OSError
[2]: https://docs.python.org/2/library/exceptions.html#exception-hierarchy